README.md

```diff
@@ -61,6 +61,7 @@ Counts expressions that branch the control flow (if-statements, loops, catch-blo
 -   function declarations (including lambda expressions)
 -   binary logical operations (like AND and OR)
 
+Note: In Java, the default label in a switch statement is counted for complexity metric.
```

Comment:
There is still no documentation that we also count structs and records with the classes metric...